### DISCUSSION TITLE
Not able to install directus 9 manually

### DISCUSSION THREAD

**Discussion by jmodanwal** [directus/directus#3619] [Archive]
Hi
I am trying to setup directus 9 manually following these https://docs.directus.io/guides/installation/manual.html

1. npm init -y
2. npm install directus
3. npm install sqlite
4. Created .env file with following variables:
PORT=8055
DB_CLIENT=sqlite3
DB_FILENAME=data.db
KEY=0707
SECRET=0707inc
STORAGE_LOCATIONS=local
--------------------------------
but when I try to run npm start it giving error:
19:51:50 💀 Database doesn't have Directus tables installed.
19:51:50 ⚠️  pino.final with prettyPrint does not support flushing

I am using node -v = 14.15.1 and npm -v = 6.14.8
What should I do ? Please let me know asap. 

Thanks.

**Comment by benhaynes** [comment]
Are you installing over an existing database with tables? This seems like a bug... might be worth opening an issue for this.

**Reply by jmodanwal** [reply]
I am doing fresh install. When I run npm start then it create db file data.db but didn't create db tables into it.

**Reply by benhaynes** [reply]
Yeah, I'd open this as an issue then so we can track it as a bug.

**Reply by jmodanwal** [reply]
okay, Thanks. Also when I try to install using CLI then it also have similar issues with sqlite db which is not able to seed the db, throws error.
What should I do? How long should I wait?

**Reply by benhaynes** [reply]
Not sure, that's a question for @rijkvanzanten ... but let's start by opening a GitHub issue (as opposed to this Discussion).

**Reply by philleepflorence** [reply]
In the meantime, try following this updated doc...I had some issues installing D9 so I updated the doc to reflect what worked for me.

You are not naming the project directus, correct?

https://github.com/philleepflorence/directus-9-suite/blob/docs/docs/guides/installation/cli.md

**Reply by rijkvanzanten** [reply]
`npm start` doesn't actually install / migrate the database. It just starts the server. You can use `npx directus init` to install the DB

**Reply by dmtkpv** [reply]
`node node_modules/directus/cli.js init` also works

**Comment by Rar9** [comment]
Why is the .env file ignored when all setting for Database are defined?

`npx directus init `
? Choose your database client (Use arrow keys)
❯ PostgreSQL / Redshift 
  CockroachDB (Beta) 
  MySQL / MariaDB / Aurora 
  SQLite 
  Microsoft SQL Server 
  Oracle Database
